katharsis-core/src/main/java/io/katharsis/queryParams/DefaultQueryParamsSerializer.java

```diff
@@ -1,14 +1,11 @@
-package io.katharsis.client.internal;
+package io.katharsis.queryParams;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00120)